apps/mobile-wallet/app.config.js

```diff
@@ -59,12 +59,6 @@ export default {
         foregroundImage: './assets/adaptive-icon.png',
         backgroundColor: '#000000'
       },
-      permissions: [
-        'android.permission.FOREGROUND_SERVICE',
-        'android.permission.FOREGROUND_SERVICE_DATA_SYNC',
-        'android.permission.START_FOREGROUND_SERVICES_FROM_BACKGROUND', // To potentially fix crash due to ForegroundServiceStartNotAllowedException
```

Comment:
Are we sure it wasn't preventing the crashes?

Reply:
We removed the background action for keeping wallet connect connection live in the background. So it's not needed.

Reply:
We have no access to Sentry since we haven't solved the quota limit due to spamming still. So we can't know if it's solved.